Podfile

```diff
@@ -5,7 +5,7 @@ platform :ios, '7.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 
-pod 'toxcore-ios', :podspec => 'https://raw.githubusercontent.com/Chuongv/toxcore-ios/mannol/toxcore-ios.podspec' 
+pod 'toxcore-ios', :podspec => 'https://raw.githubusercontent.com/Chuongv/toxcore-ios/0.1.7-new-av/toxcore-ios.podspec' 
```

Comment:
My only concern though is that building the framework from source takes 10+ mins from what I remembered. But anyways I went ahead and added back libvpx.